util-images/probes/pkg/kubeclient/metrics.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubeclient
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/perf-tests/util-images/probes/pkg/common"
+	"k8s.io/perf-tests/util-images/probes/pkg/util"
+)
+
+var (
+	inClusterAPIServerRequestLatency = prometheus.NewHistogram(prometheus.HistogramOpts{
```

Comment:
If I understand correctly, you are specifying 3 prometheus histograms here, but you are not using them anywhere? I don't see any ".Observe" calls. I guess this can be used if you also want some client-side metrics, but if not I think you can get rid of it.